dunwoody_disaster/views/MapScreen.py

```diff
@@ -119,8 +127,8 @@ def available_rooms(self) -> list:
 
     def pixmap(self) -> QPixmap:
         rooms = self.coordinates()
-        result = QPixmap(self.image).scaledToWidth(750)  # original size 1024x1024
-
+        # Scale the map image
+        result = QPixmap(self.image).scaledToWidth(1270)  # original size 1024x1024
```

Comment:
Changing this is going to throw off the coordinates. I should really make a function for automatically scaling.

Reply:
Ah I see, do you have an idea on how we should do it then?